pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/OfflineClusterIntegrationTest.java

```diff
@@ -318,6 +317,37 @@ public void testTimeFunc()
     assertEquals(todayStr, expectedTodayStr);
   }
 
+  @Test
+  public void testLiteralOnlyFunc()
+      throws Exception {
+    long currentTsMin = System.currentTimeMillis();
+    String sqlQuery = "SELECT 1, now() as currentTs, 'abc', toDateTime(now(), 'yyyy-MM-dd z') as today";
```

Comment:
Can you also add a function column without alias?

Reply:
Current Calcite Compiler already computed the results and set as a literal. so the column name will be the literal computed. I think we need to change that logic and do the evaluation here. That can be done in the next PR.